.github/workflows/ci.yml

```diff
@@ -7,13 +7,26 @@ on:
     branches: ["main"]
 
 jobs:
-  native:
+  generateMatrix:
+    name: "Generate matrix from cabal"
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout base repo
+        uses: actions/checkout@v2
```

Comment:
`sed -e s/@v2/@v3/1`

Reply:
I'm gonna use dependabot for these details :p